src/main/java/edu/nextstep/racingcarwinner/domain/Name.java

```diff
@@ -0,0 +1,32 @@
+package edu.nextstep.racingcarwinner.domain;
+
+
+import java.util.Objects;
+
+public class Name {
+    private final String name;
+
+    public Name(String name) {
+        if (name == null || name.isEmpty() || name.trim().isEmpty()) {
+            throw new IllegalArgumentException("자동차 이름은 필수 입니다.");
+        }
```

Comment:
```suggestion
        if (name == null || name.trim().isEmpty()) {
            throw new IllegalArgumentException("자동차 이름은 필수 입니다.");
        }
```
이렇게만 해도 될 것 같아요!
여기도 마찬가지로 validate method로 분리해주면 좋을 것 같아요!